### DISCUSSION TITLE
Question: Where to find docs of ScrollView in WinUI 2.5?

### DISCUSSION THREAD

**Discussion by d2phap** [microsoft/microsoft-ui-xaml#7516] [Q&A]
I found in WinUI 2.5, there is a "ScrollView" control which is much better than "ScrollViewer" control.
But I cannot find the documentation about it.

The ScrollView's APIs are marked as "for evaluation purpose", and the `ChangeView()` does not exists any more!
How to programmatically scroll the content of "ScrollView"?

![image](https://user-images.githubusercontent.com/3154213/88473020-e2a23700-cf4b-11ea-8b72-ac85f82a7aa1.png)

Thanks

**Comment by Felix-Dev** [comment]
@d2phap There are no public docs on docs.microsoft.com yet because the new `ScrollView` control is still in *preview* and might not yet be finalized for an initial V1 release. 

You are already using the correct method above to programmatically scroll content in the new `ScrollView` control. ScrollView.ScrollTo() is the new method to use where previously ScrollViewer.ChangeView() was used. See the `ScrollView` spec [here](https://github.com/microsoft/microsoft-ui-xaml-specs/blob/a9945d28c66c939e43f1bcbbf9a29a73d4b041e3/active/ScrollingView/ScrollingView.md#using-view-centric-methods-and-events-in-code).

**Comment by d2phap** [comment]
Thanks @Felix-Dev !
I tried this preview control and the alignment when zooming out of existing ScrollViewer is fixed.
The new control is more fancy and its APIs are more semantic and flexible.

However, the API `ScrollView.ScrollTo()` is not working properly with `ManipulationDelta` event.
The distance while panning by mouse is very very short (my code I tried posted at #2990).
Hopefully, this control will be finalized soon!

- Is the team going to add this new control to WinUI 3 as well? I can't find it there.
- I want to suggest to **customize (or disable) the key to enable ZoomMode**. Currently, it's `Ctrl`, and there is no way to change to other keys, or disable it (I tried with ScrollViewer).

**Comment by Felix-Dev** [comment]
The new `ScrollView` control will definitely be added to WinUI 3+. Now, whether it will make it into WinUI 3.0 or be ready only for subsequent WinUI 3.x releases can probably be answered better by @RBrid.
